ui/app/pages/swaps/view-quote/view-quote.js

```diff
@@ -503,7 +519,7 @@ export default function ViewQuote() {
             'view-quote__warning-wrapper--thin': !isShowingWarning,
           })}
         >
-          {!showInsufficientWarning && viewQuotePriceDifferenceComponent}
+          {showPriceDifferenceWarning && viewQuotePriceDifferenceComponent}
```

Comment:
I think this should be `!showInsufficientWarning && showPriceDifferenceWarning`. The current UX is that if the user does not have sufficient funds, we don't show them anything about price difference as it is irrelevant. We should maintain that UX.